lab-michael/README.md

```diff
@@ -0,0 +1,27 @@
+## TCP SIMPLE CHAT SERVER
+
+### Project Details:
+
+This is a simple chat application that will allow users to connect and gives
+the ability to change their nickname, send a direct message to other users, and
+send a message to the channel where everyone can see it. It also comes with nice
+messages for more user help.
+
+### Instructions and Usage:
+Here are some step-by-step directions.
+1. Open up a terminal where the server file is located and type:```node server.js ```
+2. Open a second terminal and type: ```localhost 3000```
```

Comment:
Is this a linux specific command? I think you need an additional command in there and then localhost 3000 are cli arguments to that command.